courier/persistence.go

```diff
@@ -7,13 +7,10 @@ import (
 	"context"
 
 	"github.com/gofrs/uuid"
-	"github.com/pkg/errors"
 
 	"github.com/ory/x/pagination/keysetpagination"
 )
 
-var ErrQueueEmpty = errors.New("queue is empty")
```

Comment:
Not sure why we had an error here, if we would always gracefully handle it. An empty list is still a valid list, and can be iterated over.